### PR TITLE
highlight code snippets in Readme

### PR DESCRIPTION
**Pull request by Borda**
just added the right language association to the code samples in Readme so it is easier to read

cc: @str0zzapreti

### CODE REVIEW

**Comment by str0zzapreti** [comment]
Ha, alright

File: README.md [hunk 1]
```diff
@@ -75,15 +75,15 @@ precedence over options specified in one of these config files. Here are some
...
+```toml
 [tool.pytest.ini_options]
 retries = 2
 retry_delay = 0.5
 cumulative_timing = false
 ```
 
 _config.ini/tox.ini_
-```
+```toml
```
Comment (str0zzapreti) [inline README.md:86]: Is there association for .ini? That would be the correct value here if so. Otherwise, I suppose toml works.

Comment (Borda) [reply README.md:86]:
```suggestion
```ini
```

Comment (Borda) [reply README.md:86]:
```suggestion
```ini
```

**Review by str0zzapreti — APPROVED** [review]

**Comment by str0zzapreti** [comment]
Thank you @Borda

**Pull request merged**